scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2279,6 +2286,10 @@ def _get_containers(
                 # add this container to the list of results
                 result[cid] = {
                     "name": cname,
+                    # construct a pod name that is similar to what we see from labels in a docker runtime
+                    "metrics_name": construct_metrics_container_name(
+                        cname, pod_name, pod_namespace, k8s_info.get("pod_uid", "")
```

Comment:
So if pod_uid is not available, metric name will be ``...__0``, is that desired?

Reply:
Not exactly desired, but if we are missing expected parts of the `k8s_info` I don't think there is any amount of massaging that will make it look correct. It's really a case we should never hit but maybe something like `<unknown>` would be better, just to make it clear something is wrong.

Reply:
Yeah, agreed, unknown would be better.